apps/web/src/components/NavMenusSection/data.tsx

```diff
@@ -0,0 +1,102 @@
+import type { ReactNode } from "react";
+import {
+  BookOpenIcon,
+  ChartBarIcon,
+  CommandLineIcon,
+  Squares2X2Icon,
+} from "@heroicons/react/24/solid";
+
+import { env } from "~/env.mjs";
+import EthereumIcon from "~/icons/ethereum.svg";
+import {
+  buildBlobStatsRoute,
+  buildBlockStatsRoute,
+  buildBlocksRoute,
+  buildTransactionStatsRoute,
+  buildTransactionsRoute,
+  buildBlobsRoute,
+} from "~/utils";
+
+function resolveApiUrl(): string {
+  if (env.NEXT_PUBLIC_NETWORK_NAME === "mainnet") {
+    return "https://api.blobscan.com";
+  }
+
+  return `https://api.${env.NEXT_PUBLIC_NETWORK_NAME}.blobscan.com`;
+}
+
+export type MenuItem = {
+  label: string;
+  href: string;
+  icon: ReactNode;
+  type: "single";
+};
+
+export type ExpandibleMenuItem = {
+  label: string;
+  icon: ReactNode;
+  items: {
+    label: string;
+    href: string;
+  }[];
+  type: "expandible";
+};
+
+export const MENU_DATA: Array<MenuItem | ExpandibleMenuItem> = [
```

Comment:
Let's rename this to `MENU_ITEMS`

Reply:
Done [3c39bab](https://github.com/Blobscan/blobscan/pull/469/commits/3c39babd2d9bd6f2262f68223123c7d822b3d0f2)